tests/modules/malt/run/main.nf

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { UNZIP } from '../../../../modules/unzip/main.nf' addParams( options: [:] )
+include { MALT_BUILD } from  '../../../../modules/malt/build/main.nf' addParams( options: [:] )
+include { MALT_RUN } from '../../../../modules/malt/run/main.nf' addParams( options: [:] )
+
+workflow test_malt_run {
+
+    fastas = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+    gff = file(params.test_data['sarscov2']['genome']['genome_gff3'], checkIfExists: true)
+    seq_type = "DNA"
+    map_db = file("https://software-ab.informatik.uni-tuebingen.de/download/megan6/megan-nucl-Jan2021.db.zip", checkIfExists: true)
```

Comment:
I don't know this tool... but would it be possible to create a small database based on toy data? 

If not, or if complicated, the CI seems still be able to cope with this in a reasonable time, so maybe not so important.

Reply:
In this case not possible unfortunately, it's a custom SQL version of NCBI taxdump (AFAIK)...

The module won't be updated that regularly though so it shouldn't block stuff too often.

Reply:
Ok, let's go then!